indico/util/mdx_latex.py

```diff
@@ -253,33 +268,37 @@ def tolatex(self, ournode):
         elif ournode.tag == 'q':
             buffer += "`%s'" % subcontent.strip()
         elif ournode.tag == 'p':
-            buffer += '\n%s\n' % subcontent.strip()
+            # do not escape if it is latex already
+            if contains_supported_latex_symbols(subcontent):
```

Comment:
What I meant was that it's weird that at this point we're not sure whether the text here is going to be LaTeX or not. It's not so much about the quality of the check (which is better now, thanks!) as it is about the need to do it.